.github/workflows/ci.yml

```
@@ -1,8 +1,5 @@
 name: CI
-on:
-  push:
-    branches:
-      - "**"
+on: [push, pull_request]
```

In general, I'm on the fence about how it's best to configure this. With your configuration organization members always create two **identical** runs cause one is created for the push event and the other one is usually created for the pull_request event.

In XState, we've mitigated this by configuring it like this:
```suggestion
on:
  push:
    branches: main
  pull_request
```

It's not quite ideal from my PoV though because I'd really like for any push by organization members to kick off those checks :shrug:

---

Ahhh yeah that makes sense.